client/src/components/HomePage/style.css

```diff
@@ -3,13 +3,18 @@
 .home__image-container {
   position: relative;
 }
-
+.homepage {
+  background-color: white;
+}
 .home__image {
   width: 100%;
   height: 550px;
   opacity: 1;
 }
 
+.react-reveal{
+  animation-duration: 1500ms !important;
```

Comment:
@HemaSAli  It looks like you can pass a `duration` prop to the Fade component directly (found out [here](https://www.react-reveal.com/docs/props/)) instead of using the !important style here.  I'm not sure if it matters much - maybe it will let you have more control, by having different Fades use different duration values.